set.mm

```diff
@@ -157042,6 +157051,16 @@ notation is used by Donald Knuth for iterated exponentiation (_Science_
     UQGIAUMBILNOMPBVAQZUOVCUTVHVJUNVBGHBVAUMERMVJUSVFUKVGEVJURVEUPEVJUQVDGIBVAU
     MIRNNBVAGHLOPUMVAAJCSTBJCSTUAUB $.
 
+  ${
+    binom2d.1 $e |- ( ph -> A e. CC ) $.
+    binom2d.2 $e |- ( ph -> B e. CC ) $.
+    $( Deduction form of binom2.  (Contributed by Igor Ieskov, 14-Dec-2023.) $)
```

Comment:
```suggestion
    $( Deduction form of ~ binom2 .  (Contributed by Igor Ieskov,
       14-Dec-2023.) $)
```

Reply:
Fixed in https://github.com/metamath/set.mm/pull/4926/commits/7a68c47ad0d3d7310d3af9b732063ce7ae518b01